ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
+```
+# apt-get update
+```
+
+Now carry on with `apt-get install` followed by the command.
+
+```
+# apt-get install libayatana-ido3-0.4-dev
+# apt-get install libayatana-indicator3-dev
+# apt-get install libcanberra-dev
+# apt-get install libgnome-desktop-3-dev
+# apt-get install libgtk-3-dev
+# apt-get install liblightdm-gobject-1-dev
+# apt-get install valac
+# apt-get install xvfb
+# apt-get install fonts-cantarell
+# apt-get install fonts-cabin
+# apt-get install gnome-settings-daemon
+```
+
+Install libpam-x2go dependencies, use the same command `apt-get install` followed by the dependencies:
+
+```
+# apt-get install libgtest-dev
+# apt-get install libpam0g-dev
+# apt-get install libssh-dev 
+# apt-get install cdbs 
+# apt-get install socat
+# apt-get install pyhoca-cli
+```
  
-Install the lightdm-remote-session-x2go dependencies, use the same command **"`apt-get install`"** followed by the dependencies:
+Install the lightdm-remote-session-x2go dependencies, use the same command `apt-get install` followed by the dependencies:
 
--	**`dh-apparmor`**
+```
+# apt-get install dh-apparmor
+```
  
-install remote-logon-service dependencies, use the same command **"`apt-get install`"** followed by the dependencies:
+install remote-logon-service dependencies, use the same command `apt-get install` followed by the dependencies:
 
--	**`dbus-test-runner`**  
--	**`libdbustest1-dev`**
--	**`libgcrypt11-dev`** 
--	**`libjson-glib-dev`** 
--	**`libnm-glib-dev`**
--	**`libsoup2.4-dev`**
+```
+# apt-get install dbus-test-runner  
+# apt-get install libdbustest1-dev
+# apt-get install libgcrypt11-dev 
+# apt-get install libjson-glib-dev 
+# apt-get install libnm-glib-dev
+# apt-get install libsoup2.4-dev
+```
 
 ## Stage five
 
-Install the devscripts, using same command **"`apt-get install`"** followed by the dependence:
+Install the devscripts, using same command `apt-get install` followed by the dependence:
 
--	**`apt-get install devscripts`**
+```
+# apt-get install devscripts
+```
 
-Now navigate to each repository folder that was created, in order to open the folders you will need to type **"`cd`"** in front of each folder, though to go back one folder you will need to type **"`cd ..`"**
+Now navigate to each repository folder that was created, in order to open the folders you will need to type `cd` in front of each folder, though to go back one folder you will need to type `cd ..`
 
 Type following:
 
--	**`cd arctica-greeter`**
+```
+cd arctica-greeter
+```
 
 Now you are in the folder here you will need to run the command:
 
--	**`debuild –uc –us`**
+```
+debuild –uc –us
+```
 
-exit the folder by typing **"`cd ..`"** and navigate to the other folders and do the same
+exit the folder by typing `cd ..` and navigate to the other folders and do the same
 
--	**`cd libpam-x2go`**
--	**`cd lightdm-remote-session-x2go`**
--	**`cd remote-logon-service`**
+```
+cd libpam-x2go
+cd lightdm-remote-session-x2go
+cd remote-logon-service
+```
```

Comment:
the debuild -uc -us commands for the other folders are missing. Please add that, e.g.

```
$ cd libpam-x2go && debuild -uc -us && cd ..
```

Then you don't have to mention the "cd .." in the below text anymore.